src/Microsoft.AspNet.Mvc.Core/ApplicationModels/DefaultActionModelBuilder.cs

```diff
@@ -20,10 +22,12 @@ namespace Microsoft.AspNet.Mvc.ApplicationModels
     public class DefaultActionModelBuilder : IActionModelBuilder
     {
         private readonly AuthorizationOptions _authorizationOptions;
+        private readonly CorsOptions _corsOptions;
 
-        public DefaultActionModelBuilder(IOptions<AuthorizationOptions> options)
+        public DefaultActionModelBuilder(IOptions<AuthorizationOptions> options, IOptions<CorsOptions> corsOptions)
         {
             _authorizationOptions = options?.Options ?? new AuthorizationOptions();
+            _corsOptions = corsOptions?.Options ?? new CorsOptions();
```

Comment:
I'm not sure how this got here for the `_authorizationOptions`.. Why is this here? is this in case no one ever runs a setup on the `CorsOptions`?

Can we rename `options` -> `authorizationOptions` and add a comment?